js/gemini.js

```diff
@@ -1317,7 +1317,7 @@ module.exports = class gemini extends Exchange {
     }
 
     nonce () {
-        return this.milliseconds ();
+        return this.seconds ();
```

Comment:
@samgermain Here and there, this issue pops out again, and we had `seconds` in the past, but I think it increases the likelihood of this problem happen.

Also, Gemini strongly recommends using `milliseconds`
![image](https://user-images.githubusercontent.com/43336371/207289182-5b669f64-9aa3-4481-9d77-c7b589a721ab.png)
https://docs.gemini.com/rest-api/#timestamps

So I think we should not change it but ask the user to try to sync the clock.

Reply:
That's weird because the server time that they return is in seconds `1670883809`

Reply:
The user has said "I synchronize my PC clock with NTP time server." in their issue

Reply:
@samgermain, yeah but as I said in the past we had seconds and the problem was more frequent so I don't think we should change it

Reply:
@carlosmiei It doesn't seem random to me. Every time I try to use gemini I get the same error, and the solution is to switch to seconds